src/pages/Checkout-two-step.page.js

```diff
@@ -0,0 +1,71 @@
+import { BaseSwagLabPage } from './BaseSwagLab.page';
+
+
+export class CheckoutTwoStepPage extends BaseSwagLabPage {
+    url = '/checkout-step-two.html';  
+
+    get inventoryItems() {return this.page.locator('.cart_item')};
+         
+    get inventoryItemName() { return this.page.locator('.inventory_item_name')};
+
+    get inventoryDescription() { return this.page.locator('.inventory_item_desc')};
+
+    get inventoryPrice() {return this.page.locator('.inventory_item_price')}; 
+
+    get itemsSubTotal() { return this.page.locator('.summary_subtotal_label')};
+
+    get tax() { return this.page.locator('.summary_tax_label')};
+
+    get itemsTotal() {return this.page.locator('.summary_total_label')}; 
+
+   async getCheckoutItemsLst() {
+      return this.getItemsListData(await this.inventoryItems.all());
+  }
+  
+  async getItemsListData(items) {
```

Comment:
Is this method reusable? I am not sure we will use it anywhere but in getCheckoutItemsList. Lets move this logic to the getCheckoutItemsList method.